<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Luiz  Gadelha</title>
    <meta name="author" content="Luiz  Gadelha" />
    <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
" />
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light" />

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>⚛️</text></svg>">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://lgadelha.github.io/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark" />

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item active">
                <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a>
              </li>
              
              <!-- Blog -->
              <li class="nav-item ">
                <a class="nav-link" href="/blog/">blog</a>
              </li>

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/projects/">projects</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/publications/">publications</a>
              </li>
              <li class="nav-item dropdown ">
                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus</a>
                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
                  <a class="dropdown-item" href="/publications/">publications</a>
                  <div class="dropdown-divider"></div>
                  <a class="dropdown-item" href="/projects/">projects</a>
                </div>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/teaching/">teaching</a>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- about.html -->
      <div class="post">
        <header class="post-header">
          <h1 class="post-title">
           <span class="font-weight-bold">Luiz</span>  Gadelha
          </h1>
          <p class="desc">Postdoctoral Researcher</p>
        </header>

        <article>
          <div class="profile float-right">
<figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/prof_pic-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/prof_pic-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/prof_pic-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid z-dept-1 rounded" src="/assets/img/prof_pic.jpg" alt="prof_pic.jpg">

  </picture>

</figure>

            <div class="address">
              <p>FUSION Research Group</p> <p>University of Jena</p> <p>Leutragraben 1, Room 21S03</p> <p>07743 Jena - Germany</p>

            </div>
          </div>

          <div class="clearfix">
            <p>I’m a postdoctoral researcher in the Cluster of Excellence Balance of the Microverse at the Friedrich-Schiller-University Jena in Germany working on research data management. I’m currently on leave as a researcher at the National Laboratory for Scientific Computing (LNCC) in Brazil. I received my D.Sc. degree in Computer and Systems Engineering from the Federal University of Rio de Janeiro (UFRJ), Brazil. I’ve been involved in the research and development of parallel and distributed scientific workflow management systems and scientific databases. I’ve participated in research projects in the bioinformatics and biodiversity application areas. My main research interests are scientific workflows,
computational reproducibility, and parallel computing. I’m a member of the Brazilian Computer Society (SBC) and the Association for Computing Machinery (ACM).</p>

          </div>

          <!-- News -->          
          <div class="news">
            <h2>news</h2>
            <div class="table-responsive">
              <table class="table table-sm table-borderless"> 
                <tr>
                  <th scope="row">Jan 15, 2016</th>
                  <td>
                    A simple inline announcement with Markdown emoji! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20">
 
                  </td>
                </tr> 
                <tr>
                  <th scope="row">Nov 7, 2015</th>
                  <td>
                    <a class="news-title" href="/news/announcement_2/">A long announcement with details</a> 
                  </td>
                </tr> 
                <tr>
                  <th scope="row">Oct 22, 2015</th>
                  <td>
                    A simple inline announcement.
 
                  </td>
                </tr> 
              </table>
            </div> 
          </div>

          <!-- Selected papers -->
          <div class="publications">
            <h2>selected publications</h2>
            <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">eScience</abbr></div>

        <!-- Entry bib key -->
        <div id="Mondelli2021" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Capturing and Semantically Describing Provenance to Tell the Story of R Scripts</div>
          <!-- Author -->
          <div class="author">Mondelli, Maria Luiza, Samuel, Sheeba, Konig-Ries, Birgitta, and <em>Gadelha, Luiz</em>
                
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>In 2021 IEEE 17th International Conference on eScience (eScience)</em> Sep 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a href="https://ieeexplore.ieee.org/document/9582412/" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">WIREs DMKD</abbr></div>

        <!-- Entry bib key -->
        <div id="Gadelha2020a" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">A survey of biodiversity informatics: Concepts, practices, and challenges</div>
          <!-- Author -->
          <div class="author">
                  <em>Gadelha, Luiz</em>, Siracusa, Pedro C., Dalcin, Eduardo Couto, Silva, Luís Alexandre Estevão, Augusto, Douglas A., Krempser, Eduardo, Affe, Helen Michelle, Costa, Raquel Lopes, Mondelli, Maria Luiza, Meirelles, Pedro Milet, Thompson, Fabiano, Chame, Marcia, Ziviani, Artur, and Siqueira, Marinez Ferreira
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery</em> Nov 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://onlinelibrary.wiley.com/doi/10.1002/widm.1394" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="https://wires.onlinelibrary.wiley.com/doi/epdf/10.1002/widm.1394" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The unprecedented size of the human population, along with its associated economic activities, has an ever-increasing impact on global environments. Across the world, countries are concerned about the growing resource consumption and the capacity of ecosystems to provide resources. To effectively conserve biodiversity, it is essential to make indicators and knowledge openly available to decision-makers in ways that they can effectively use them. The development and deployment of tools and techniques to generate these indicators require having access to trustworthy data from biological collections, field surveys and automated sensors, molecular data, and historic academic literature. The transformation of these raw data into synthesized information that is fit for use requires going through many refinement steps. The methodologies and techniques applied to manage and analyze these data constitute an area usually called biodiversity informatics. Biodiversity data follow a life cycle consisting of planning, collection, certification, description, preservation, discovery, integration, and analysis. Researchers, whether producers or consumers of biodiversity data, will likely perform activities related to at least one of these steps. This article explores each stage of the life cycle of biodiversity data, discussing its methodologies, tools, and challenges. This article is categorized under: Algorithmic Development &gt; Biological Data Mining.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Sci. Rep.</abbr></div>

        <!-- Entry bib key -->
        <div id="Siracusa2020" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">New perspectives on analysing data from biological collections based on social network analytics</div>
          <!-- Author -->
          <div class="author">Siracusa, Pedro C., 
                  <em>Gadelha, Luiz</em>, and Ziviani, Artur
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Scientific Reports</em> Dec 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a href="http://www.nature.com/articles/s41598-020-60134-y" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="https://www.nature.com/articles/s41598-020-60134-y.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">FGCS</abbr></div>

        <!-- Entry bib key -->
        <div id="Ocana2020" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">BioinfoPortal: A scientific gateway for integrating bioinformatics applications on the Brazilian national high-performance computing network</div>
          <!-- Author -->
          <div class="author">Ocaña, Kary A.C.S., Galheigo, Marcelo, Osthoff, Carla, 
                  <em>Gadelha, Luiz</em>, Porto, Fabio, Gomes, Antônio Tadeu A., Oliveira, Daniel, and Vasconcelos, Ana Tereza
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Future Generation Computer Systems</em> Jun 2020
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a href="https://linkinghub.elsevier.com/retrieve/pii/S0167739X19318230" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">ER Workshops</abbr></div>

        <!-- Entry bib key -->
        <div id="Mondelli2019" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Exploring Reproducibility and FAIR Principles in Data Science Using Ecological Niche Modeling as a Case Study</div>
          <!-- Author -->
          <div class="author">Mondelli, Maria Luiza, Townsend Peterson, A., and <em>Gadelha, Luiz</em>
                
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            Jun 2019
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a href="http://link.springer.com/10.1007/978-3-030-34146-6_3" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">PeerJ</abbr></div>

        <!-- Entry bib key -->
        <div id="Mondelli2018a" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">BioWorkbench: a high-performance framework for managing and analyzing bioinformatics experiments</div>
          <!-- Author -->
          <div class="author">Mondelli, Maria Luiza, Magalhães, Thiago, Loss, Guilherme, Wilde, Michael, Foster, Ian, Mattoso, Marta, Katz, Daniel, Barbosa, Helio, Vasconcelos, Ana Tereza R., Ocaña, Kary, and <em>Gadelha, Luiz</em>
                
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>PeerJ</em> Aug 2018
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://peerj.com/articles/5551" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="https://peerj.com/articles/5551.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Advances in sequencing techniques have led to exponential growth in biological data, demanding the development of large-scale bioinformatics experiments. Because these experiments are computation- and data-intensive, they require high-performance computing techniques and can benefit from specialized technologies such as Scientific Workflow Management Systems and databases. In this work, we present BioWorkbench, a framework for managing and analyzing bioinformatics experiments. This framework automatically collects provenance data, including both performance data from workflow execution and data from the scientific domain of the workflow application. Provenance data can be analyzed through a web application that abstracts a set of queries to the provenance database, simplifying access to provenance information. We evaluate BioWorkbench using three case studies: SwiftPhylo, a phylogenetic tree assembly workflow; SwiftGECKO, a comparative genomics workflow; and RASflow, a RASopathy analysis workflow. We analyze each workflow from both computational and scientific domain perspectives, by using queries to a provenance and annotation database. Some of these queries are available as a pre-built feature of the BioWorkbench web application. Through the provenance data, we show that the framework is scalable and achieves high-performance, reducing up to 98% of the case studies execution time. We also show how the application of machine learning techniques can enrich the analysis process.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">PeerJ</abbr></div>

        <!-- Entry bib key -->
        <div id="Costa2017" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">GeNNet: an integrated platform for unifying scientific workflows and graph databases for transcriptome data analysis</div>
          <!-- Author -->
          <div class="author">Costa, Raquel L., 
                  <em>Gadelha, Luiz</em>, Ribeiro-Alves, Marcelo, and Porto, Fábio
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>PeerJ</em> Jul 2017
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://peerj.com/articles/3509" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="https://peerj.com/articles/3509.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>There are many steps in analyzing transcriptome data, from the acquisition of raw data to the selection of a subset of representative genes that explain a scientific hypothesis. The data produced can be represented as networks of interactions among genes and these may additionally be integrated with other biological databases, such as Protein-Protein Interactions, transcription factors and gene annotation. However, the results of these analyses remain fragmented, imposing difficulties, either for posterior inspection of results, or for meta-analysis by the incorporation of new related data. Integrating databases and tools into scientific workflows, orchestrating their execution, and managing the resulting data and its respective metadata are challenging tasks. Additionally, a great amount of effort is equally required to run in-silico experiments to structure and compose the information as needed for analysis. Different programs may need to be applied and different files are produced during the experiment cycle. In this context, the availability of a platform supporting experiment execution is paramount. We present GeNNet, an integrated transcriptome analysis platform that unifies scientific workflows with graph databases for selecting relevant genes according to the evaluated biological systems. It includes GeNNet-Wf, a scientific workflow that pre-loads biological data, pre-processes raw microarray data and conducts a series of analyses including normalization, differential expression inference, clusterization and gene set enrichment analysis. A user-friendly web interface, GeNNet-Web, allows for setting parameters, executing, and visualizing the results of GeNNet-Wf executions. To demonstrate the features of GeNNet, we performed case studies with data retrieved from GEO, particularly using a single-factor experiment in different analysis scenarios. As a result, we obtained differentially expressed genes for which biological functions were analyzed. The results are integrated into GeNNet-DB, a database about genes, clusters, experiments and their properties and relationships. The resulting graph database is explored with queries that demonstrate the expressiveness of this data model for reasoning about gene interaction networks. GeNNet is the first platform to integrate the analytical process of transcriptome data with graph databases. It provides a comprehensive set of tools that would otherwise be challenging for non-expert users to install and use. Developers can add new functionality to components of GeNNet. The derived data allows for testing previous hypotheses about an experiment and exploring new ones through the interactive graph database environment. It enables the analysis of different data on humans, rhesus, mice and rat coming from Affymetrix platforms. GeNNet is available as an open source platform at https://github.com/raquele/GeNNet and can be retrieved as a software container with the command docker pull quelopes/gennet.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Database</abbr></div>

        <!-- Entry bib key -->
        <div id="Meirelles2015" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">BaMBa: towards the integrated management of Brazilian marine environmental data</div>
          <!-- Author -->
          <div class="author">Meirelles, Pedro Milet, 
                  <em>Gadelha, Luiz</em>, Francini-Filho, Ronaldo Bastos, Leão, Rodrigo de Moura, Amado-Filho, Gilberto Menezes, Bastos, Alex Cardoso, Paranhos, Rodolfo Pinheiro da Rocha, Rezende, Carlos Eduardo, Swings, Jean, Siegle, Eduardo, Neto, Nils Edvin Asp, Leitão, Sigrid Neumann, Coutinho, Ricardo, Mattoso, Marta, Salomon, Paulo S., Valle, Rogério A. B., Pereira, Renato Crespo, Kruger, Ricardo Henrique, Thompson, Cristiane, and Thompson, Fabiano L.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Database</em> Jan 2015
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="http://database.oxfordjournals.org/content/2015/bav088.abstract" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="https://academic.oup.com/database/article-pdf/doi/10.1093/database/bav088/7299083/bav088.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>A new open access database, Brazilian Marine Biodiversity (BaMBa) (https://marinebiodiversity.lncc.br), was developed in order to maintain large datasets from the Brazilian marine environment. Essentially, any environmental information can be added to BaMBa. Certified datasets obtained from integrated holistic studies, comprising physical–chemical parameters, -omics, microbiology, benthic and fish surveys can be deposited in the new database, enabling scientific, industrial and governmental policies and actions to be undertaken on marine resources. There is a significant number of databases, however BaMBa is the only integrated database resource both supported by a government initiative and exclusive for marine data. BaMBa is linked to the Information System on Brazilian Biodiversity (SiBBr, http://www.sibbr.gov.br/) and will offer opportunities for improved governance of marine resources and scientists’ integration. Database URL: http://marinebiodiversity.lncc.br</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">IPAW</abbr></div>

        <!-- Entry bib key -->
        <div id="Gadelha2014" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Applying Provenance to Protect Attribution in Distributed Computational Scientific Experiments</div>
          <!-- Author -->
          <div class="author">
                  <em>Gadelha, Luiz</em>, and Mattoso, Marta
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            Jan 2015
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a href="http://dx.doi.org/10.1007/978-3-319-16462-5_11" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
            <a href="https://link.springer.com/content/pdf/10.1007/978-3-319-16462-5_11.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">DPD</abbr></div>

        <!-- Entry bib key -->
        <div id="Gadelha2012" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">MTCProv: a practical provenance query framework for many-task scientific computing</div>
          <!-- Author -->
          <div class="author">
                  <em>Gadelha, Luiz</em>, Wilde, Michael, Mattoso, Marta, and Foster, Ian
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>Distributed and Parallel Databases</em> Aug 2012
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a href="http://link.springer.com/10.1007/s10619-012-7104-4" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          
        </div>
      </div>
</li>
</ol>
          </div>

          <!-- Social -->
          <div class="social">
            <div class="contact-icons">
            <a href="mailto:%6C%75%69%7A.%67%61%64%65%6C%68@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://orcid.org/0000-0002-8122-9522" title="ORCID" target="_blank" rel="noopener noreferrer"><i class="ai ai-orcid"></i></a>
            <a href="https://scholar.google.com/citations?user=gWzznREAAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a>
            <a href="https://github.com/lgadelha" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a>
            <a href="https://www.fusion.uni-jena.de/people/details/luiz-gadelha" title="Work" target="_blank" rel="noopener noreferrer"><i class="fas fa-briefcase"></i></a>
            <a href="https://gitlab.com/lgadelha" title="GitLab" target="_blank" rel="noopener noreferrer"><i class="fab fa-gitlab"></i></a>
            <a href="http://www.informatik.uni-trier.de/~ley/pers/hd/g/Gadelha_Jr=:Luiz_M=_R=" title="DBLP" target="_blank" rel="noopener noreferrer"><i class="ai ai-dblp"></i></a>
            <a href="https://lgadelha.github.io/feed.xml" title="RSS Feed"><i class="fas fa-rss-square"></i></a>
            
            </div>

            <div class="contact-note">
              You can even add a little note about which of these is the best way to reach you.

            </div>
            
          </div>
        </article>

</div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2022 Luiz  Gadelha. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="noopener noreferrer">Unsplash</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

