<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Luiz Gadelha</title> <meta name="author" content="Luiz Gadelha"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://lgadelha.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?6185d15ea1982787ad7f435576553d64"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Luiz </span>Gadelha</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people</a> </li> <li class="nav-item dropdown active"> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus<span class="sr-only">(current)</span></a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item active" href="/publications/">publications</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/projects/">projects</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/blog/">blog</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Sci. Data</abbr></div> <div id="wilkinsonApplyingFAIRPrinciples2025" class="col-sm-8"> <div class="title">Applying the FAIR Principles to Computational Workflows</div> <div class="author"> Sean R. Wilkinson, Meznah Aloqalaa, Khalid Belhajjame, and <span class="more-authors" title="click to view 20 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '20 more authors' ? 'Michael R. Crusoe, Bruno de Paula Kinoshita, Luiz Gadelha, Daniel Garijo, Ove Johan Ragnar Gustafsson, Nick Juty, Sehrish Kanwal, Farah Zaib Khan, Johannes Köster, Karsten Peters-von Gehlen, Line Pouchard, Randy K. Rannow, Stian Soiland-Reyes, Nicola Soranzo, Shoaib Sufi, Ziheng Sun, Baiba Vilne, Merridee A. Wouters, Denis Yuen, Carole Goble' : '20 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">20 more authors</span> </div> <div class="periodical"> <em>Scientific Data</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Recent trends within computational and data sciences show an increasing recognition and adoption of computational workflows as tools for productivity, reproducibility, and democratized access to platforms and processing know-how. As digital objects to be shared, discovered, and reused, computational workflows benefit from the FAIR principles, which stand for Findable, Accessible, Interoperable, and Reusable. The Workflows Community Initiative’s FAIR Workflows Working Group (WCI-FW), a global and open community of researchers and developers working with computational workflows across disciplines and domains, has systematically addressed the application of both FAIR data and software principles to computational workflows. We present our recommendations with commentary that reflects our discussions and justifies our choices and adaptations. Like the software and data principles on which they are based, these are offered to workflow users and authors, workflow management system developers, and providers of workflow services as guide rails for adoption and fodder for discussion. Workflows are becoming more prevalent as documented, automated instruments for data analysis, data collection, AI-based predictions, and simulations. The FAIR recommendations for workflows that we propose in this paper will maximize their value as research assets and facilitate their adoption by the wider community.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PeerJ CS</abbr></div> <div id="ahmedEvaluatingMethodReproducibility2025" class="col-sm-8"> <div class="title">Evaluating the method reproducibility of deep learning models in biodiversity research</div> <div class="author"> Waqas Ahmed, Vamsi Krishna Kommineni, Birgitta König-Ries, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jitendra Gaikwad, Luiz Gadelha, Sheeba Samuel' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>PeerJ Computer Science</em>, Feb 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://peerj.com/articles/cs-2618/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://peerj.com/articles/cs-2618.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Artificial intelligence (AI) is revolutionizing biodiversity research by enabling advanced data analysis, species identification, and habitats monitoring, thereby enhancing conservation efforts. Ensuring reproducibility in AI-driven biodiversity research is crucial for fostering transparency, verifying results, and promoting the credibility of ecological findings. This study investigates the reproducibility of deep learning (DL) methods within the biodiversity research. We design a methodology for evaluating the reproducibility of biodiversity-related publications that employ DL techniques across three stages. We define ten variables essential for method reproducibility, divided into four categories: resource requirements, methodological information, uncontrolled randomness, and statistical considerations. These categories subsequently serve as the basis for defining different levels of reproducibility. We manually extract the availability of these variables from a curated dataset comprising 100 publications identified using the keywords provided by biodiversity experts. Our study shows that a dataset is shared in 50% of the publications; however, a significant number of the publications lack comprehensive information on deep learning methods, including details regarding randomness.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BMC Bioinf.</abbr></div> <div id="costaHIHISIVDatabaseGene2024" class="col-sm-8"> <div class="title">HIHISIV: a database of gene expression in HIV and SIV host immune response</div> <div class="author"> Raquel L. Costa, <em>Luiz Gadelha</em>, Mirela D’arc, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Marcelo Ribeiro-Alves, David L. Robertson, Jean-Marc Schwartz, Marcelo A. Soares, Fábio Porto' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>BMC Bioinformatics</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05740-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://bmcbioinformatics.biomedcentral.com/counter/pdf/10.1186/s12859-024-05740-7.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Abstract In the battle of the host against lentiviral pathogenesis, the immune response is crucial. However, several questions remain unanswered about the interaction with different viruses and their influence on disease progression. The simian immunodeficiency virus (SIV) infecting nonhuman primates (NHP) is widely used as a model for the study of the human immunodeficiency virus (HIV) both because they are evolutionarily linked and because they share physiological and anatomical similarities that are largely explored to understand the disease progression. The HIHISIV database was developed to support researchers to integrate and evaluate the large number of transcriptional data associated with the presence/absence of the pathogen (SIV or HIV) and the host response (NHP and human). The datasets are composed of microarray and RNA-Seq gene expression data that were selected, curated, analyzed, enriched, and stored in a relational database. Six query templates comprise the main data analysis functions and the resulting information can be downloaded. The HIHISIV database, available at  https://hihisiv.github.io , provides accurate resources for browsing and visualizing results and for more robust analyses of pre-existing data in transcriptome repositories.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Metabolites</abbr></div> <div id="zulfiqarImplementationFAIRPractices2024a" class="col-sm-8"> <div class="title">Implementation of FAIR Practices in Computational Metabolomics Workflows—A Case Study</div> <div class="author"> Mahnoor Zulfiqar, Michael R. Crusoe, Birgitta König-Ries, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Christoph Steinbeck, Kristian Peters, Luiz Gadelha' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Metabolites</em>, Feb 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.mdpi.com/2218-1989/14/2/118" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://www.mdpi.com/2218-1989/14/2/118/pdf?version=1708164641" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Scientific workflows facilitate the automation of data analysis tasks by integrating various software and tools executed in a particular order. To enable transparency and reusability in workflows, it is essential to implement the FAIR principles. Here, we describe our experiences implementing the FAIR principles for metabolomics workflows using the Metabolome Annotation Workflow (MAW) as a case study. MAW is specified using the Common Workflow Language (CWL), allowing for the subsequent execution of the workflow on different workflow engines. MAW is registered using a CWL description on WorkflowHub. During the submission process on WorkflowHub, a CWL description is used for packaging MAW using the Workflow RO-Crate profile, which includes metadata in Bioschemas. Researchers can use this narrative discussion as a guideline to commence using FAIR practices for their bioinformatics or cheminformatics workflows while incorporating necessary amendments specific to their research area.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">IWSG</abbr></div> <div id="krugerGHGAArchiveSelected2024a" class="col-sm-8"> <div class="title">The GHGA Archive: Selected Updates</div> <div class="author"> Jens Krüger, Jordy Didier Orellana Figueroa, Zehra Hazal Sezer, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Thomas Jakob Zajac, Kersten Breuer, Luiz Manoel Rocha Gadelha Junior, Christoph Zwerschke, Bilge Sürün, Sven Nahnsen' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>In 16th International Workshop on Science Gateways (IWSG2024)</em>, Sep 2024 </div> <div class="periodical"> Publisher: Zenodo </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://zenodo.org/doi/10.5281/zenodo.13863209" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://zenodo.org/records/13863209/files/IWSG2024_paper_5.pdf?download=1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>The German Human Genome-Phenome Archive (GHGA) is a cross-institutional project and German National Research Data Infrastructure (NFDI) consortium for the development of a scientific gateway for secure omics data sharing based on FAIR principles to act as the German node of the federated European Genome Archive (fEGA), participating also in the European Genomics Data Infrastructure (GDI) project. Started in 2020, the GHGA is reaching its first major release milestone: having released an online science gateway for metadata browsing termed the Metadata Catalog, the next step involves the development of a portal that extends the functionality already available to also include authentication, data download (via an external tool), and data access request management: the GHGA Archive. We present here a brief overview of the new technologies and functionalities that will be available for the Archive version of the GHGA science gateway.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">J. Cheminf.</abbr></div> <div id="zulfiqarMAWReproducibleMetabolome2023a" class="col-sm-8"> <div class="title">MAW: the reproducible Metabolome Annotation Workflow for untargeted tandem mass spectrometry</div> <div class="author"> Mahnoor Zulfiqar, <em>Luiz Gadelha</em>, Christoph Steinbeck, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Maria Sorokina, Kristian Peters' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Journal of Cheminformatics</em>, Mar 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-023-00695-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://jcheminf.biomedcentral.com/counter/pdf/10.1186/s13321-023-00695-y.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Abstract Mapping the chemical space of compounds to chemical structures remains a challenge in metabolomics. Despite the advancements in untargeted liquid chromatography-mass spectrometry (LC–MS) to achieve a high-throughput profile of metabolites from complex biological resources, only a small fraction of these metabolites can be annotated with confidence. Many novel computational methods and tools have been developed to enable chemical structure annotation to known and unknown compounds such as in silico generated spectra and molecular networking. Here, we present an automated and reproducible M etabolome A nnotation W orkflow (MAW) for untargeted metabolomics data to further facilitate and automate the complex annotation by combining tandem mass spectrometry (MS 2 ) input data pre-processing, spectral and compound database matching with computational classification, and in silico annotation. MAW takes the LC-MS 2 spectra as input and generates a list of putative candidates from spectral and compound databases. The databases are integrated via the R package Spectra and the metabolite annotation tool SIRIUS as part of the R segment of the workflow (MAW-R). The final candidate selection is performed using the cheminformatics tool RDKit in the Python segment (MAW-Py). Furthermore, each feature is assigned a chemical structure and can be imported to a chemical structure similarity network. MAW is following the FAIR (Findable, Accessible, Interoperable, Reusable) principles and has been made available as the docker images, maw-r and maw-py. The source code and documentation are available on GitHub ( https://github.com/zmahnoor14/MAW ). The performance of MAW is evaluated on two case studies. MAW can improve candidate ranking by integrating spectral databases with annotation tools like SIRIUS which contributes to an efficient candidate selection procedure. The results from MAW are also reproducible and traceable, compliant with the FAIR guidelines. Taken together, MAW could greatly facilitate automated metabolite characterization in diverse fields such as clinical metabolomics and natural product discovery.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CoRDI</abbr></div> <div id="gadelhaGermanHumanGenomePhenome2023" class="col-sm-8"> <div class="title">German Human Genome-Phenome Archive in an International Context: Toward a Federated Infrastructure for Managing and Analyzing Genomics and Health Data</div> <div class="author"> <em>Luiz Gadelha</em>, and Jan Eufinger</div> <div class="periodical"> <em>Proceedings of the Conference on Research Data Infrastructure</em>, Sep 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.tib-op.org/ojs/index.php/CoRDI/article/view/394" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://www.tib-op.org/ojs/index.php/CoRDI/article/view/394/446" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>With increasing numbers of human omics data, there is an urgent need for adequate resources for data sharing while also standardizing and harmonizing data processing. As part of the National Research Data Infrastructure (NFDI), the German Human Genome-Phenome Archive (GHGA) strives to connect the data from German researchers and their institutions to the international landscape of genome research. To achieve this, GHGA partners up with international activities such as the federated European Genome-Phenome Archive (EGA) [1] and the recently funded European Genomic Data Infrastructure (GDI) project to enable participation in international studies while ensuring at the same time the proper protection of the sensitive patient data included in GHGA.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">eScience</abbr></div> <div id="gadelhaFrameworkIntegrativeFAIR2022" class="col-sm-8"> <div class="title">Toward a Framework for Integrative, FAIR, and Reproducible Management of Data on the Dynamic Balance of Microbial Communities</div> <div class="author"> <em>Luiz Gadelha</em>, Martin Hohmuth, Mahnoor Zulfiqar, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'David Schone, Sheeba Samuel, Maria Sorokina, Christoph Steinbeck, Birgitta Konig-Ries' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>In 2022 IEEE 18th International Conference on e-Science (e-Science)</em>, Oct 2022 </div> <div class="periodical"> </div> <div class="links"> <a href="https://ieeexplore.ieee.org/document/9973522/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://arxiv.org/pdf/2207.06890" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CARLA</abbr></div> <div id="ocanaParslRNASeqEfficientScalable2022" class="col-sm-8"> <div class="title">ParslRNA-Seq: An Efficient and Scalable RNAseq Analysis Workflow for Studies of Differentiated Gene Expression</div> <div class="author"> Kary Ocaña, Lucas Cruz, Micaella Coelho, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Rafael Terra, Marcelo Galheigo, Andre Carneiro, Diego Carvalho, Luiz Gadelha, Francieli Boito, Philippe Navaux, Carla Osthoff' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>In High Performance Computing</em>, Oct 2022 </div> <div class="periodical"> Series Title: Communications in Computer and Information Science </div> <div class="links"> <a href="https://link.springer.com/10.1007/978-3-031-23821-5_13" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Comp. y Sis.</abbr></div> <div id="cruzParallelPerformanceProfiling2022" class="col-sm-8"> <div class="title">Parallel Performance and I/O Profiling of HPC RNA-Seq Applications</div> <div class="author"> Lucas Cruz, Micaella Coelho, Marcelo Galheigo, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Andre Carneiro, Diego Carvalho, Luiz Gadelha, Francieli Boito, Philippe Navaux, Carla Osthoff, Kary Ocaña' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Computación y Sistemas</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a href="https://cys.cic.ipn.mx/ojs/index.php/CyS/article/view/4437" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://cys.cic.ipn.mx/ojs/index.php/CyS/article/view/4437/3471" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Biodiv. Inf.</abbr></div> <div id="Peterson2022" class="col-sm-8"> <div class="title">ENM2020: A Free Online Course and Set of Resources on Modeling Species’ Niches and Distributions | Biodiversity Informatics</div> <div class="author"> A. Townsend Peterson, Matthew Aiello-Lammens, Giuseppe Amatulli, and <span class="more-authors" title="click to view 41 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '41 more authors' ? 'Robert Anderson, Marlon Cobos, José Alexandre Diniz-Filho, Luis Escobar, Xiao Feng, Janet Franklin, Luiz Gadelha, Damien Georges, M Guéguen, Tomer Gueta, Kate Ingenloff, Scott Jarvie, Laur Jiménez, Dirk Karger, Jamie Kass, Michael Kearney, Rafael Loyola, Fernando Machado-Stredel, Enrique Martínez-Meyer, Cory Merow, Maria Luiza Mondelli, Sara Mortara, Robert Muscarella, Corinne Myers, Babak Naimi, Daniel Noesgaard, Ian Ondo, Luis Osorio-Olvera, Hannah Owens, Richard Pearson, Gonzalo Pinilla-Buitrago, Andrea Sánchez-Tapia, Erin Saupe, Wilfried Thuiller, Sara Varela, Dan Warren, John Wieczorek, Katherine Yates, Gengping Zhu, Gabriela Zuquim, Damaris Zurell' : '41 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">41 more authors</span> </div> <div class="periodical"> <em>Biodiversity Informatics</em>, Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a href="https://journals.ku.edu/jbi/article/view/15016" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://journals.ku.edu/jbi/article/view/15016/15152" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">eScience</abbr></div> <div id="Mondelli2021" class="col-sm-8"> <div class="title">Capturing and Semantically Describing Provenance to Tell the Story of R Scripts</div> <div class="author"> Maria Luiza Mondelli, Sheeba Samuel, Birgitta Konig-Ries, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Luiz Gadelha' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>In 2021 IEEE 17th International Conference on eScience (eScience)</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a href="https://ieeexplore.ieee.org/document/9582412/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">WIREs DMKD</abbr></div> <div id="Gadelha2020a" class="col-sm-8"> <div class="title">A survey of biodiversity informatics: Concepts, practices, and challenges</div> <div class="author"> <em>Luiz Gadelha</em>, Pedro C. Siracusa, Eduardo Couto Dalcin, and <span class="more-authors" title="click to view 11 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '11 more authors' ? 'Luís Alexandre Estevão Silva, Douglas A. Augusto, Eduardo Krempser, Helen Michelle Affe, Raquel Lopes Costa, Maria Luiza Mondelli, Pedro Milet Meirelles, Fabiano Thompson, Marcia Chame, Artur Ziviani, Marinez Ferreira Siqueira' : '11 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">11 more authors</span> </div> <div class="periodical"> <em>Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery</em>, Nov 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://onlinelibrary.wiley.com/doi/10.1002/widm.1394" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://wires.onlinelibrary.wiley.com/doi/epdf/10.1002/widm.1394" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>The unprecedented size of the human population, along with its associated economic activities, has an ever-increasing impact on global environments. Across the world, countries are concerned about the growing resource consumption and the capacity of ecosystems to provide resources. To effectively conserve biodiversity, it is essential to make indicators and knowledge openly available to decision-makers in ways that they can effectively use them. The development and deployment of tools and techniques to generate these indicators require having access to trustworthy data from biological collections, field surveys and automated sensors, molecular data, and historic academic literature. The transformation of these raw data into synthesized information that is fit for use requires going through many refinement steps. The methodologies and techniques applied to manage and analyze these data constitute an area usually called biodiversity informatics. Biodiversity data follow a life cycle consisting of planning, collection, certification, description, preservation, discovery, integration, and analysis. Researchers, whether producers or consumers of biodiversity data, will likely perform activities related to at least one of these steps. This article explores each stage of the life cycle of biodiversity data, discussing its methodologies, tools, and challenges. This article is categorized under: Algorithmic Development &gt; Biological Data Mining.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Gadelha2020" class="col-sm-8"> <div class="title">Gestão de Dados de Biotecnologia Marinha</div> <div class="author"> <em>Luiz Gadelha</em> </div> <div class="periodical"> <em>In Biotecnologia Marinha</em>, Nov 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="https://cienciasdomarbrasil.furg.br/images/livros/LivroBiotecnologia.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Sci. Rep.</abbr></div> <div id="Siracusa2020" class="col-sm-8"> <div class="title">New perspectives on analysing data from biological collections based on social network analytics</div> <div class="author"> Pedro C. Siracusa, <em>Luiz Gadelha</em>, and Artur Ziviani</div> <div class="periodical"> <em>Scientific Reports</em>, Dec 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="http://www.nature.com/articles/s41598-020-60134-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://www.nature.com/articles/s41598-020-60134-y.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">FGCS</abbr></div> <div id="Ocana2020" class="col-sm-8"> <div class="title">BioinfoPortal: A scientific gateway for integrating bioinformatics applications on the Brazilian national high-performance computing network</div> <div class="author"> Kary A.C.S. Ocaña, Marcelo Galheigo, Carla Osthoff, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Luiz Gadelha, Fabio Porto, Antônio Tadeu A. Gomes, Daniel Oliveira, Ana Tereza Vasconcelos' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Future Generation Computer Systems</em>, Jun 2020 </div> <div class="periodical"> </div> <div class="links"> <a href="https://linkinghub.elsevier.com/retrieve/pii/S0167739X19318230" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ER Workshops</abbr></div> <div id="Mondelli2019" class="col-sm-8"> <div class="title">Exploring Reproducibility and FAIR Principles in Data Science Using Ecological Niche Modeling as a Case Study</div> <div class="author"> Maria Luiza Mondelli, A. Townsend Peterson, and <em>Luiz Gadelha</em> </div> <div class="periodical"> <em>In Advances in Conceptual Modeling. ER 2019. Lecture Notes in Computer Science, vol. 11787</em>, Jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a href="http://link.springer.com/10.1007/978-3-030-34146-6_3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">iSys</abbr></div> <div id="Mondelli2019a" class="col-sm-8"> <div class="title">O Que os Países Escutam: Analisando a Rede de Gêneros Musicais ao Redor do Mundo</div> <div class="author"> Maria Luiza Mondelli, <em>Luiz Gadelha</em>, and Artur Ziviani</div> <div class="periodical"> <em>iSys - Revista Brasileira de Sistemas de Informação</em>, Jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a href="http://seer.unirio.br/isys/article/view/8099" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="http://seer.unirio.br/index.php/isys/article/view/8099/7927" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CCGrid</abbr></div> <div id="Ocana2019" class="col-sm-8"> <div class="title">Towards a Science Gateway for Bioinformatics: Experiences in the Brazilian System of High Performance Computing</div> <div class="author"> Kary Ocana, Marcelo Galheigo, Carla Osthoff, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Luiz Gadelha, Antonio Tadeu A. Gomes, Daniel De Oliveira, Fabio Porto, Ana Tereza Vasconcelos' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>In 2019 19th IEEE/ACM International Symposium on Cluster, Cloud and Grid Computing (CCGRID)</em>, May 2019 </div> <div class="periodical"> </div> <div class="links"> <a href="https://ieeexplore.ieee.org/document/8752906/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JHIR</abbr></div> <div id="Chame2019" class="col-sm-8"> <div class="title">SISS-Geo: Leveraging Citizen Science to Monitor Wildlife Health Risks in Brazil</div> <div class="author"> Marcia Chame, Helio J. C. Barbosa, <em>Luiz Gadelha</em>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Douglas A. Augusto, Eduardo Krempser, Livia Abdalla' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Journal of Healthcare Informatics Research</em>, Dec 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://link.springer.com/10.1007/s41666-019-00055-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>The well-being of wildlife health involves many challenges, such as monitoring the movement of pathogens; expanding health surveillance beyond humans; collecting data and extracting information to identify and predict risks; integrating specialists from different areas to handle data, species and distinct social and environmental contexts; and, the commitment to bringing relevant information to society. In Brazil, there is still the difficulty of building a mechanism that is not impaired by its large territorial extension and its poorly integrated sectoral policies. The Brazilian Wildlife Health Information System, SISS-Geo, is a platform for collaborative monitoring that intends to overcome the challenges in wildlife health. It aims integration and participation of various segments of society, encompassing: the registration of occurrences by citizen scientists; the reliable diagnosis of pathogens from the laboratory and expert networks; and computational and mathematical challenges in analytical and predictive systems, knowledge extraction, data integration and visualization, and geographic information systems. It has been successfully applied to support decision-making on recent wildlife health events, such as a Yellow Fever epizooty.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BrasNAM</abbr></div> <div id="Mondelli2018" class="col-sm-8"> <div class="title">O Que os Países Escutam: Analisando a Rede de Gêneros Musicais ao Redor do Mundo</div> <div class="author"> Maria Luiza Mondelli, <em>Luiz Gadelha</em>, and Artur Ziviani</div> <div class="periodical"> <em>In Anais do VII Brazilian Workshop on Social Network Analysis and Mining (BrasNAM 2018)</em>, Dec 2018 </div> <div class="periodical"> </div> <div class="links"> <a href="https://sol.sbc.org.br/index.php/brasnam/article/view/3586" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://sol.sbc.org.br/index.php/brasnam/article/view/3586/3545" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PeerJ</abbr></div> <div id="Mondelli2018a" class="col-sm-8"> <div class="title">BioWorkbench: a high-performance framework for managing and analyzing bioinformatics experiments</div> <div class="author"> Maria Luiza Mondelli, Thiago Magalhães, Guilherme Loss, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Michael Wilde, Ian Foster, Marta Mattoso, Daniel Katz, Helio Barbosa, Ana Tereza R. Vasconcelos, Kary Ocaña, Luiz Gadelha' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>PeerJ</em>, Aug 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://peerj.com/articles/5551" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://peerj.com/articles/5551.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>Advances in sequencing techniques have led to exponential growth in biological data, demanding the development of large-scale bioinformatics experiments. Because these experiments are computation- and data-intensive, they require high-performance computing techniques and can benefit from specialized technologies such as Scientific Workflow Management Systems and databases. In this work, we present BioWorkbench, a framework for managing and analyzing bioinformatics experiments. This framework automatically collects provenance data, including both performance data from workflow execution and data from the scientific domain of the workflow application. Provenance data can be analyzed through a web application that abstracts a set of queries to the provenance database, simplifying access to provenance information. We evaluate BioWorkbench using three case studies: SwiftPhylo, a phylogenetic tree assembly workflow; SwiftGECKO, a comparative genomics workflow; and RASflow, a RASopathy analysis workflow. We analyze each workflow from both computational and scientific domain perspectives, by using queries to a provenance and annotation database. Some of these queries are available as a pre-built feature of the BioWorkbench web application. Through the provenance data, we show that the framework is scalable and achieves high-performance, reducing up to 98% of the case studies execution time. We also show how the application of machine learning techniques can enrich the analysis process.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">FMS</abbr></div> <div id="Thompson2018" class="col-sm-8"> <div class="title">Marine Biotechnology in Brazil: Recent Developments and Its Potential for Innovation</div> <div class="author"> Fabiano Thompson, Ricardo Krüger, Cristiane C. Thompson, and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? 'Roberto G. S. Berlinck, Ricardo Coutinho, Melissa F. Landell, Mauro Pavão, Paulo A. S. Mourão, Ana Salles, Naiane Negri, Fabyano A. C. Lopes, Vitor Freire, Alexandre J. Macedo, Marcelo Maraschin, Carlos D. Pérez, Renato C. Pereira, Gandhi Radis-Baptista, Rachel P. Rezende, Wagner C. Valenti, Paulo C. Abreu, Biotecmar Network' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em>Frontiers in Marine Science</em>, Jul 2018 </div> <div class="periodical"> </div> <div class="links"> <a href="https://www.frontiersin.org/article/10.3389/fmars.2018.00236/full" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://www.frontiersin.org/articles/10.3389/fmars.2018.00236/pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CARLA</abbr></div> <div id="Sanchez-Tapia2018" class="col-sm-8"> <div class="title">Model-R: A Framework for Scalable and Reproducible Ecological Niche Modeling</div> <div class="author"> Andrea Sánchez-Tapia, Marinez Ferreira Siqueira, Rafael Oliveira Lima, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Felipe Sodré M. Barros, Guilherme M. Gall, Luiz Gadelha, Luís Alexandre E. Silva, Carla Osthoff' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>In High Performance Computing: 4th Latin American Conference, CARLA 2017. Communications in Computer and Information Science, vol. 796</em>, Jul 2018 </div> <div class="periodical"> </div> <div class="links"> <a href="http://link.springer.com/10.1007/978-3-319-73353-1_15" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PeerJ</abbr></div> <div id="Costa2017" class="col-sm-8"> <div class="title">GeNNet: an integrated platform for unifying scientific workflows and graph databases for transcriptome data analysis</div> <div class="author"> Raquel L. Costa, <em>Luiz Gadelha</em>, Marcelo Ribeiro-Alves, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Fábio Porto' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>PeerJ</em>, Jul 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://peerj.com/articles/3509" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://peerj.com/articles/3509.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>There are many steps in analyzing transcriptome data, from the acquisition of raw data to the selection of a subset of representative genes that explain a scientific hypothesis. The data produced can be represented as networks of interactions among genes and these may additionally be integrated with other biological databases, such as Protein-Protein Interactions, transcription factors and gene annotation. However, the results of these analyses remain fragmented, imposing difficulties, either for posterior inspection of results, or for meta-analysis by the incorporation of new related data. Integrating databases and tools into scientific workflows, orchestrating their execution, and managing the resulting data and its respective metadata are challenging tasks. Additionally, a great amount of effort is equally required to run in-silico experiments to structure and compose the information as needed for analysis. Different programs may need to be applied and different files are produced during the experiment cycle. In this context, the availability of a platform supporting experiment execution is paramount. We present GeNNet, an integrated transcriptome analysis platform that unifies scientific workflows with graph databases for selecting relevant genes according to the evaluated biological systems. It includes GeNNet-Wf, a scientific workflow that pre-loads biological data, pre-processes raw microarray data and conducts a series of analyses including normalization, differential expression inference, clusterization and gene set enrichment analysis. A user-friendly web interface, GeNNet-Web, allows for setting parameters, executing, and visualizing the results of GeNNet-Wf executions. To demonstrate the features of GeNNet, we performed case studies with data retrieved from GEO, particularly using a single-factor experiment in different analysis scenarios. As a result, we obtained differentially expressed genes for which biological functions were analyzed. The results are integrated into GeNNet-DB, a database about genes, clusters, experiments and their properties and relationships. The resulting graph database is explored with queries that demonstrate the expressiveness of this data model for reasoning about gene interaction networks. GeNNet is the first platform to integrate the analytical process of transcriptome data with graph databases. It provides a comprehensive set of tools that would otherwise be challenging for non-expert users to install and use. Developers can add new functionality to components of GeNNet. The derived data allows for testing previous hypotheses about an experiment and exploring new ones through the interactive graph database environment. It enables the analysis of different data on humans, rhesus, mice and rat coming from Affymetrix platforms. GeNNet is available as an open source platform at https://github.com/raquele/GeNNet and can be retrieved as a software container with the command docker pull quelopes/gennet.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">SBBD Demo</abbr></div> <div id="Mondelli2016a" class="col-sm-8"> <div class="title">HPSW-Prof: A Provenance-Based Framework for Profiling High Performance Scientific Workflows</div> <div class="author"> Maria Luiza Mondelli, Matheus Tonelli Souza, Kary Ocaña, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Ana Tereza R. Vasconcelos, Luiz Gadelha' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>In Proc. of Satellite Events of the 31st Brazilian Symposium on Databases (SBBD 2016)</em>, Jul 2016 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BreSci</abbr></div> <div id="Mondelli2016" class="col-sm-8"> <div class="title">Integrating Scientific Workflows with Scientific Gateways: A Bioinformatics Experiment in the Brazilian National High-Performance Computing Network</div> <div class="author"> Maria Luiza Mondelli, Marcelo Galheigo, Vivian Medeiros, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Bruno Fernandes Bastos, Antonio Tadeu Azevedo Gomes, Ana Tereza R Vasconcelos, Luiz Gadelha' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>In X Brazilian e-Science Workshop. Anais do XXXVI Congresso da Sociedade Brasileira de Computação</em>, Jul 2016 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Database</abbr></div> <div id="Meirelles2015" class="col-sm-8"> <div class="title">BaMBa: towards the integrated management of Brazilian marine environmental data</div> <div class="author"> Pedro Milet Meirelles, <em>Luiz Gadelha</em>, Ronaldo Bastos Francini-Filho, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Rodrigo de Moura Leão, Gilberto Menezes Amado-Filho, Alex Cardoso Bastos, Rodolfo Pinheiro da Rocha Paranhos, Carlos Eduardo Rezende, Jean Swings, Eduardo Siegle, Nils Edvin Asp Neto, Sigrid Neumann Leitão, Ricardo Coutinho, Marta Mattoso, Paulo S. Salomon, Rogério A. B. Valle, Renato Crespo Pereira, Ricardo Henrique Kruger, Cristiane Thompson, Fabiano L. Thompson' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Database</em>, Jan 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://database.oxfordjournals.org/content/2015/bav088.abstract" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://academic.oup.com/database/article-pdf/doi/10.1093/database/bav088/7299083/bav088.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="abstract hidden"> <p>A new open access database, Brazilian Marine Biodiversity (BaMBa) (https://marinebiodiversity.lncc.br), was developed in order to maintain large datasets from the Brazilian marine environment. Essentially, any environmental information can be added to BaMBa. Certified datasets obtained from integrated holistic studies, comprising physical–chemical parameters, -omics, microbiology, benthic and fish surveys can be deposited in the new database, enabling scientific, industrial and governmental policies and actions to be undertaken on marine resources. There is a significant number of databases, however BaMBa is the only integrated database resource both supported by a government initiative and exclusive for marine data. BaMBa is linked to the Information System on Brazilian Biodiversity (SiBBr, http://www.sibbr.gov.br/) and will offer opportunities for improved governance of marine resources and scientists’ integration. Database URL: http://marinebiodiversity.lncc.br</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PLOS ONE</abbr></div> <div id="Meirelles2015a" class="col-sm-8"> <div class="title">Baseline Assessment of Mesophotic Reefs of the Vitória-Trindade Seamount Chain Based on Water Quality, Microbial Diversity, Benthic Cover and Fish Biomass Data</div> <div class="author"> Pedro M. Meirelles, Gilberto M. Amado-Filho, Guilherme H. Pereira-Filho, and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? 'Hudson T. Pinheiro, Rodrigo L. Moura, Jean-Christophe Joyeux, Eric F. Mazzei, Alex C. Bastos, Robert A. Edwards, Elizabeth Dinsdale, Rodolfo Paranhos, Eidy O. Santos, Tetsuya Iida, Kazuyoshi Gotoh, Shota Nakamura, Tomoo Sawabe, Carlos E. Rezende, Luiz Gadelha, Ronaldo B. Francini-Filho, Cristiane Thompson, Fabiano L. Thompson' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em>PLOS ONE</em>, Jun 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="http://dx.plos.org/10.1371/journal.pone.0130084" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0130084&amp;representation=PDF" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">IPAW</abbr></div> <div id="Gadelha2014" class="col-sm-8"> <div class="title">Applying Provenance to Protect Attribution in Distributed Computational Scientific Experiments</div> <div class="author"> <em>Luiz Gadelha</em>, and Marta Mattoso</div> <div class="periodical"> <em>In Provenance and Annotation of Data and Processes. IPAW 2014. Lecture Notes in Computer Science, vol. 8628</em>, Jun 2015 </div> <div class="periodical"> </div> <div class="links"> <a href="http://dx.doi.org/10.1007/978-3-319-16462-5_11" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://link.springer.com/content/pdf/10.1007/978-3-319-16462-5_11.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Data Sci. J.</abbr></div> <div id="Zorrilla2014" class="col-sm-8"> <div class="title">Conceptual View Representation of the Brazilian Information System on Antarctic Environmental Research</div> <div class="author"> R. Zorrilla, M. Poltosi, <em>L. Gadelha</em>, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'F. Porto, A. Moura, A. Dalto, H. P. Lavrado, Y. Valentin, M. Tenório, E. Xavier' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Data Science Journal</em>, Oct 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://www.jstage.jst.go.jp/article/dsj/13/0/13_IFPDA-04/_article" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://www.jstage.jst.go.jp/article/dsj/13/0/13_IFPDA-04/_pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">BreSci</abbr></div> <div id="Gadelha2014a" class="col-sm-8"> <div class="title">SiBBr: Uma Infraestrutura para Coleta, Integração e Análise de Dados sobre a Biodiversidade Brasileira</div> <div class="author"> <em>Luiz Gadelha</em>, Pedro Guimarães, Ana Maria Moura, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Debora P. Drucker, Eduardo Dalcin, Guilherme Gall, Jurandir Tavares, Daniele Palazzi, Maira Poltosi, Fabio Porto, Francisco Moura, Wagner Vieira Leo' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>In VIII Brazilian e-Science Workshop (BRESCI 2014). Proc. XXXIV Congress of the Brazilian Computer Society</em>, Oct 2014 </div> <div class="periodical"> </div> <div class="links"> <a href="https://sol.sbc.org.br/index.php/bresci/article/view/10477" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://sol.sbc.org.br/index.php/bresci/article/view/10477/10344" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">EDBT/ICDT WS</abbr></div> <div id="Gadelha2013" class="col-sm-8"> <div class="title">Provenance traces of the Swift parallel scripting system</div> <div class="author"> <em>Luiz Gadelha</em>, Michael Wilde, Marta Mattoso, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Ian Foster' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>In Proceedings of the Joint EDBT/ICDT 2013 Workshops on - EDBT ’13</em>, Mar 2013 </div> <div class="periodical"> </div> <div class="links"> <a href="http://dl.acm.org/citation.cfm?id=2457317.2457374" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">PESC-UFRJ</abbr></div> <div id="GadelhaJr.2012" class="col-sm-8"> <div class="title">Gerência de Proveniência em Workflows Científicos Paralelos e Distribuídos</div> <div class="author"> <em>L. Gadelha</em> </div> <div class="periodical"> <em>Universidade Federal do Rio de Janeiro</em>, Mar 2012 </div> <div class="periodical"> </div> <div class="links"> <a href="https://www.cos.ufrj.br/index.php/pt-BR/publicacoes-pesquisa/details/15/2331" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://www.cos.ufrj.br/uploadfile/1347903453.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ISEI</abbr></div> <div id="Gadelha2012a" class="col-sm-8"> <div class="title">Scalable and Provenance-Enabled Scientific Workflows for Predicting Distribution of Species.</div> <div class="author"> <em>L. Gadelha</em>, S. Stanzani, P. Corrêa, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'E. Dalcin, C. Gomes, L. Sato, M. Siqueira' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>In Proc. 8th International Conference on Ecological Informatics (ISEI 2012)</em>, Mar 2012 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">DPD</abbr></div> <div id="Gadelha2012" class="col-sm-8"> <div class="title">MTCProv: a practical provenance query framework for many-task scientific computing</div> <div class="author"> <em>Luiz Gadelha</em>, Michael Wilde, Marta Mattoso, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Ian Foster' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Distributed and Parallel Databases</em>, Aug 2012 </div> <div class="periodical"> </div> <div class="links"> <a href="http://link.springer.com/10.1007/s10619-012-7104-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">TaPP</abbr></div> <div id="Gadelha2011a" class="col-sm-8"> <div class="title">Provenance Query Patterns for Many-Task Scientific Computing</div> <div class="author"> <em>L. Gadelha</em>, M. Mattoso, M. Wilde, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'I. Foster' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>In 3rd USENIX Workshop on the Theory and Practice of Provenance (TaPP 2011)</em>, Aug 2011 </div> <div class="periodical"> </div> <div class="links"> <a href="http://www.usenix.org/events/tapp11/tech/final_files/GadelhaUpdate.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">HPCDB</abbr></div> <div id="Gadelha2011" class="col-sm-8"> <div class="title">Exploring provenance in high performance scientific computing</div> <div class="author"> <em>Luiz Gadelha</em>, Michael Wilde, Marta Mattoso, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Ian Foster' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>In Proc. of the 1st Annual Workshop on High Performance Computing meets Databases - HPCDB ’11</em>, Nov 2011 </div> <div class="periodical"> </div> <div class="links"> <a href="http://dl.acm.org/citation.cfm?id=2125636.2125643" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">FGCS</abbr></div> <div id="GadelhaJr.2011" class="col-sm-8"> <div class="title">Provenance management in Swift</div> <div class="author"> <em>Luiz Gadelha</em>, Ben Clifford, Marta Mattoso, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Michael Wilde, Ian Foster' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Future Generation Computer Systems</em>, Jun 2011 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://www.sciencedirect.com/science/article/pii/S0167739X1000083X" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>The Swift parallel scripting language allows for the specification, execution and analysis of large-scale computations in parallel and distributed environments. It incorporates a data model for recording and querying provenance information. In this article we describe these capabilities and evaluate the interoperability with other systems through the use of the Open Provenance Model. We describe Swift’s provenance data model and compare it to the Open Provenance Model. We also describe and evaluate activities performed within the Third Provenance Challenge, which consisted of implementing a specific scientific workflow, capturing and recording provenance information of its execution, performing provenance queries, and exchanging provenance information with other systems. Finally, we propose improvements to both the Open Provenance Model and Swift’s provenance system.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">IPAW</abbr></div> <div id="Gadelha2010" class="col-sm-8"> <div class="title">Towards a Threat Model for Provenance in e-Science</div> <div class="author"> <em>L. Gadelha</em>, M. Mattoso, M. Wilde, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'I. Foster' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>In Provenance and Annotation of Data and Processes - The Third Provenance and Annotation Workshop (IPAW 2010)</em>, Jun 2010 </div> <div class="periodical"> </div> <div class="links"> <a href="http://link.springer.com/chapter/10.1007/978-3-642-17819-1_32" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ANL-TR</abbr></div> <div id="Gadelha2010a" class="col-sm-8"> <div class="title">Provenance Management in Swift with Implementation Details</div> <div class="author"> <em>L. Gadelha</em>, B. Clifford, M. Mattoso, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'M. Wilde, I. Foster.' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> Jun 2010 </div> <div class="periodical"> </div> <div class="links"> <a href="http://www.mcs.anl.gov/papers/TM-311-final.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> </div> </div> </li> </ol> <h2 class="bibliography">2008</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">eScience</abbr></div> <div id="GadelhaJr2008" class="col-sm-8"> <div class="title">Kairos: An Architecture for Securing Authorship and Temporal Information of Provenance Data in Grid-Enabled Workflow Management Systems</div> <div class="author"> <em>Luiz Gadelha</em>, and Marta Mattoso</div> <div class="periodical"> <em>In 2008 IEEE Fourth International Conference on eScience</em>, Dec 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://ieeexplore.ieee.org/lpdocs/epic03/wrapper.htm?arnumber=4736871" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Secure provenance techniques are essential in generating trustworthy provenance records, where one is interested in protecting their integrity, confidentiality, and availability. In this work, we suggest an architecture to provide protection of authorship and temporal information in grid-enabled provenance systems. It can be used in the resolution of conflicting intellectual property claims, and in the reliable chronological reconstitution of scientific experiments. We observe that some techniques from public key infrastructures can be readily applied for this purpose. We discuss the issues involved in the implementation of such architecture and describe some experiments realized with the proposed techniques.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CCGrid</abbr></div> <div id="Cruz2008" class="col-sm-8"> <div class="title">A Lightweight Middleware Monitor for Distributed Scientific Workflows</div> <div class="author"> Sergio Manuel Serra Cruz, Fabricio Nogueira Silva, <em>Luiz Gadelha</em>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Maria Claudia Reis Cavalcanti, Maria Luiza M. Campos, Marta Mattoso' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>In 2008 Eighth IEEE International Symposium on Cluster Computing and the Grid (CCGRID)</em>, May 2008 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://ieeexplore.ieee.org/lpdocs/epic03/wrapper.htm?arnumber=4534285" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Monitoring the execution of distributed tasks within the workflow execution is not easy and is frequently controlled manually. This work presents a lightweight middleware monitor to design and control the parallel execution of tasks from a distributed scientific workflow. This middleware can be connected into a workflow management system. This middleware implementation is evaluated with the Kepler workflow management system, by including new modules to control and monitor the distributed execution of the tasks. These middleware modules were added to a bio informatics workflow to monitor parallel BLAST executions. Results show potential to high performance process execution while preserving the original features of the workflow.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2007</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">MGC</abbr></div> <div id="Gadelha2007" class="col-sm-8"> <div class="title">On the management of grid credentials</div> <div class="author"> <em>Luiz Gadelha</em>, and Bruno Schulze</div> <div class="periodical"> <em>In Proceedings of the 5th international workshop on Middleware for grid computing held at the ACM/IFIP/USENIX 8th International Middleware Conference - MGC ’07</em>, Nov 2007 </div> <div class="periodical"> </div> <div class="links"> <a href="http://dl.acm.org/citation.cfm?id=1376849.1376859" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2006</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">WCGA</abbr></div> <div id="Gadelha2006" class="col-sm-8"> <div class="title">Specification of a MyProxy Plugin for Mozilla</div> <div class="author"> <em>L. Gadelha</em> </div> <div class="periodical"> <em>In Proc. IV Workshop de Computação em Grid e Aplicações (WCGA 2006)</em>, Nov 2006 </div> <div class="periodical"> </div> <div class="links"> <a href="http://bibliotecadigital.sbc.org.br/download.php?paper=2286" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2001</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">SCCC</abbr></div> <div id="Gadelha2001" class="col-sm-8"> <div class="title">An efficient strategy for word-cycle completion in finitely presented groups</div> <div class="author"> <em>L. Gadelha</em>, and M. Ayala-Rincon</div> <div class="periodical"> <em>In SCCC 2001. 21st International Conference of the Chilean Computer Science Society</em>, Nov 2001 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://ieeexplore.ieee.org/lpdocs/epic03/wrapper.htm?arnumber=972635" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>We present an efficient strategy for the application of the inference rules of a completion procedure for finitely presented groups. This procedure has been proposed by Cremanns and Otto and uses a combinatorial structure called word-cycle. Our strategy is complete in the sense that a set of persistent word-cycles can be used to solve the reduced word problem of the finitely presented group given as input. This procedure has been implemented in order to experiment with various examples of groups</p> </div> </div> </div> </li></ol> <h2 class="bibliography">1998</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Revista SCCC</abbr></div> <div id="Gadelha1998" class="col-sm-8"> <div class="title">Some Applications of (Semi-)Decision Algorithms for Presburger Arithmetic in Automated Deduction based on Rewriting Techniques</div> <div class="author"> <em>Luiz Gadelha</em>, and Mauricio Ayala-Rincón</div> <div class="periodical"> <em>La Revista de La Sociedad Chilena de Ciencia de La Computación</em>, Nov 1998 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">1997</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CLEI</abbr></div> <div id="Gadelha1997" class="col-sm-8"> <div class="title">Applications of Decision Algorithms for Presburger Arithmetic in Rewrite Automated Deduction</div> <div class="author"> <em>L. Gadelha</em>, and M. Ayala-Rincón</div> <div class="periodical"> <em>In Proc. XXIII Latin American Conference on Informatics (CLEI)</em>, Nov 1997 </div> <div class="periodical"> </div> <div class="links"> <a href="http://citeseerx.ist.psu.edu/viewdoc/download;jsessionid=B2552C9F9A4D2998F68207217096A970?doi=10.1.1.51.9473&amp;rep=rep1&amp;type=pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2025 Luiz Gadelha. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?9b43d6e67ddc7c0855b1478ee4c48c2d" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>